modules/dkan/dkan_datastore/src/Helper/Node.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Dkan\Datastore\Helper;
+
+class Node {
+  /**
+   * Gets nid using uuid.
+   */
+  public static function getNid($uuid) {
```

Comment:
`entity_uuid_load()` function exists if you want to use that instead, but that does run hooks on the entity load.